po/common/es.po

```diff
@@ -489,6 +489,10 @@ msgctxt "delete_confirmation"
 msgid "This will delete your user details and anonymise all of your contributions. Please re-enter your user name to confirm."
 msgstr "Esto eliminará sus datos de usuario y anonimizará todas sus contribuciones. Vuelva a ingresar su nombre de usuario para confirmar."
 
+msgctxt "description"
+msgid "Descripton"
```

Comment:
small typo, msgid should always be the same as the English translation

```suggestion
msgid "Description"
```

Reply:
sorry 😅, fixed the typo and changed the en.po file